jest.setup.js

```diff
@@ -14,7 +14,7 @@ global.TextEncoder = TextEncoder;
 global.TextDecoder = TextDecoder;
 
 Vue.config.productionTip = false;
-Vue.use(i18n, { store: { dispatch() {} } });
```

Comment:
Made this change in regards to https://github.com/rancher/dashboard/pull/10923#discussion_r1588528484